.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: cpp
+
+addons:
+  apt:
+    packages:
+    - swig
+    - python-dev
+    - libguile-dev
+    - octave-headers
+    - cmake
+
+script:
+  - ./autogen.sh --no-configure
+  - ./configure --prefix=$HOME/.local --enable-shared --enable-maintainer-mode --enable-cxx
```

Comment:
Since the makefiles support building in a subdirectory, that would be good to test:
```
mkdir build
cd build
../configure --prefix=$HOME/.local --enable-shared --enable-maintainer-mode --enable-cxx
make -j2
make install
```
and so on.